### PR TITLE
labeler ice discovery refactor

### PR DESCRIPTION
**Pull request by mibac138**
Supersedes #767, adds more verbose error when failing to create a new issue

### CODE REVIEW

File: labeler/src/main.rs [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-
-    issue_list
-}
-
 fn main() {
     let config = github::Config::from_env().unwrap();
 
-    let mut unlabeled_issue_list = issue_list();
+    let mut unlabeled_issue_list = glacier::discover("./ices").unwrap().into_iter().map(|ice| ice.id()).collect::<Vec<_>>();
+    unlabeled_issue_list.sort();
```
Comment (JohnTitor) [inline labeler/src/main.rs:9]:
```suggestion
    unlabeled_issue_list.sort_unstable();
```

I'd prefer `sort_unstable` as it's faster on the primitive types.

File: labeler/Cargo.toml [hunk 2]
```diff
@@ -11,3 +11,4 @@ once_cell = "1"
 regex = "1"
```
Comment (JohnTitor) [inline labeler/Cargo.toml:11]: Now we can remove `regex` dependency.

**Comment by JohnTitor** [comment]
I'm going to merge #791 first to fix CI failure, note that it will cause merge conflicts.

File: labeler/Cargo.toml (hunk 2 above)
Comment (mibac138) [reply labeler/Cargo.toml:11]: It's still used in https://github.com/rust-lang/glacier/blob/master/labeler/src/github.rs#L114-L138

Comment (JohnTitor) [reply labeler/Cargo.toml:11]: Oh, seems so. My bad!

**Pull request merged**